docs/topics/windows.md

```diff
@@ -385,30 +385,32 @@ It will also create a working Kubernetes client config file in `_output/<dnspref
 Get the paths to `azuredeploy.json` and `azuredeploy.parameters.json` from the last step, and pass them into `az group deployment create --name <name for deployment> --resource-group <resource group name> --template-file <...azuredeploy.json> --parameters <...azuredeploy.parameters.json>`
 
 ```console
-$ az group deployment create --name plangk8swin1-deploy --resource-group k8s-win1 --template-file "./_output/plangk8swin1/azuredeploy.json" --parameters "./_output/plangk8swin1/azuredeploy.parameters.json"
+$ az group deployment create --name k8s-win1-deploy --resource-group k8s-win1 --template-file "./_output/wink8s1/azuredeploy.json" --parameters "./_output/wink8s1/azuredeploy.parameters.json"
 ```
 
 After several minutes, it will return the list of resources created in JSON. Look for `masterFQDN`.
 
 ```json
       "masterFQDN": {
         "type": "String",
-        "value": "plangk8swin1.westus2.cloudapp.azure.com"
+        "value": "wink8s1.westus2.cloudapp.azure.com"
       },
 ```
 
+A dnsPrefix which forms part of the the hostname for your cluster (e.g. staging, prodwest, blueberry). The DNS prefix must be unique so pick a random name. In case you have not modified the dns prefix, it might error out.

```

Comment:
This isn't a full sentence, and I'm not sure what it relates to. Is this necessary here?

Reply:
DnsPrefix has to be unique as this is global resource and can get into errors if some other user has it under his/her name.
It is more about letting the user know to make it unique.  Wasn't sure how to add this so took a cue from the other documentation.

Reply:
```suggestion
The DNS prefix provided is used to generate the hostname of the cluster (e.g. staging, prodwest, blueberry), and must be unique for each cluster deployment.
```

Reply:
@mboersma thoughts?

Reply:
That fixes it. 👍

Reply:
@munishm if you're ok with Cécile's change above, we're ok with this PR. Thanks again!